homework-5/script-1.js

```diff
@@ -0,0 +1,86 @@
+//Вивести в console числа, від 10 до 25 з кроком 0.5 (10 й 25 включно)
+console.log("Number from 10 to 25 inclusive with step 0.5:");
+for (let i = 10; i <= 25; i += 0.5) {
+    console.log(i);
+}
+
+//Запитати та зберегти число введене із prompt
+const number = +prompt("Enter please number");
+
+//Визначати чи це число просте
+let i = 2;
+const limit = Math.sqrt(number);
+if (number > 1) {
+    while (i <= limit) {
+        if (number % i !== 0) {
+            console.log(`The number ${number} is prime`);
+            break;
+        }
+        console.log(`The number ${number} isn't prime`);
+        i++;
+    }
+} else {
+    console.log("The number isn't prime");
+}
+
+/*Запитати та зберегти суму покупки введене із prompt. Обрахувати знижку та повідомити
+ користувачу значення знижки. Якщо сума покупки до 100 UAН - знижка 3%; якщо сума покупки
+до 200 UAH - знижка 5%; якщо сума покупки перевищує 200 UAH - знижка 7% */
+const purchase = +prompt("Enter please sum of purchase (only number) UAH");
+let discount;
+if (purchase <= 100) {
+    discount = (0.03 * purchase).toFixed(2);
```

Comment:
Its better to store discounts as a constants to avoid "magical" numbers in the code